katacomb/requirements.txt

```diff
@@ -15,7 +15,6 @@ docutils
 enum34
```

Comment:
Can probably get rid of this (unless it's required by a dependency). Also ipaddress, singledispatch, futures, subprocess32 (they're all backports of Python 3 features to Python 2).

Reply:
Excellent- fewer requirements. Done.